mlflow/transformers.py

```diff
@@ -1320,28 +1320,8 @@ class _TransformersWrapper:
     def __init__(self, pipeline, inference_config=None):
         self.pipeline = pipeline
         self.inference_config = inference_config
-        if inference_config:
-            self._validate_inference_config_keys(pipeline, inference_config)
         self._conversation = None
-
-    @staticmethod
-    def _validate_inference_config_keys(pipeline, inference_config):
```

Comment:
This does nothing. The inference overrides are arguments to the wrapped `generate()` method (implicit call to the pipeline object) and are not associated with the pipeline object itself.